Session-2.0.kt

```diff
@@ -0,0 +1,50 @@
+import java.util.ArrayList
+
+fun main(args: Array<String>) {
+
+    val streetCap = 8; // Capacity of street
+    val countOfAllCars = 25; //Number of cars
+    var drivingCars: Int = 0 // Number of cars that want to drive
+    var allCars = ArrayList<Car>(); // Array of cars

```

Comment:
I just saw that this might be due to a comment I made in #42. My idea is to have three classes:

* `Car`
* something for the road network that does the actual simulation
* a third class with a `main` method that creates the cars and starts the simulation